tss.el

```diff
@@ -533,7 +533,20 @@
                (tss--trace "Got response from server.")
                tss--server-response))))))
 
-(defun* tss--sync-server (&key waitsec path buff)
+(defun* tss--sync-server (&key waitsec buff path
+                               source linecount)
+  "Sync current source with tss service. Default to 15 waiti,
```

Comment:
I'm sorry to reply very late to your contacts and confuse you by my laziness...
It means "wait index" as loop counter like `(loop for i from 1 to ...)`.